tests/sklearn/test_sklearn_models.py

```diff
@@ -1171,8 +1177,7 @@ def check_rounding_consistency(
         mp_context.setenv("TREES_USE_ROUNDING", "0")
 
         # Check that rounding is disabled
-        rounding_disabled = os.environ.get("TREES_USE_ROUNDING") == "0"
-        assert rounding_disabled
+        assert os.environ.get("TREES_USE_ROUNDING") == "0", "'TREES_USE_ROUNDING' is not disabled"
```

Comment:
Didn't we say we remove those env variable?

Reply:
Ah yeah my bad. Let's not mix everything in one PR.

Reply:
When did we decide that?
I'm fine if this is done in another PR but would prefer to not have to rely on env vars.